app.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+// list of variables used for
+var cookieCounter = 0;
```

Comment:
It's a great idea to group global variables up at the top like this. 

That said, what is cookieCounter for?